scripts/priceBoundPSMDeployUSDC.ts

```diff
@@ -0,0 +1,167 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import config from './Config';
+import { getImpersonatedSigner } from '@test/helpers';
+
+/// ---------------------------------------------------------------- ///
+///                                                                  ///
+///                Price Bound Peg Stability Module                  ///
+///                                                                  ///
+/// ---------------------------------------------------------------- ///
+
+/// Description: This module is used to manage the stability of the peg.
+
+/// Steps:
+///  0 - Deploy USDC/VOLT PriceBoundPegStabilityModule
+///  1 - Pause Redemptions on the VOLT PriceBoundPegStabilityModule
+///  2 - Move VOLT into the PriceBoundPegStabilityModule to allow trades
+
+/// This contract needs no roles in the system, just funding to allow swaps
+
+/// ---------------------------------------------------------------- ///
+
+const {
+  CORE,
+  USDC,
+  VOLT,
+  PRICE_BOUND_PSM,
+  VOLT_FUSE_PCV_DEPOSIT,
+  MINT_FEE_BASIS_POINTS,
+  REDEEM_FEE_BASIS_POINTS,
+  PROTOCOL_MULTISIG_ADDRESS,
+  ORACLE_PASS_THROUGH_ADDRESS
+} = config;
+
+const usdcReservesThreshold = ethers.utils.parseEther('10000000000'); /// this is greater than the trillions because USDC only has 6 decimals
+const mintLimitPerSecond = ethers.utils.parseEther('10000');
```

Comment:
Nit: Would recommend adding a easily human readable comment for these values. e.g. `/// 10k`